test/integrations/destinations/campaign_manager/dataDelivery/oauth.ts

```diff
@@ -389,14 +389,14 @@ export const v1oauthScenarios: ProxyV1TestData[] = [
     },
     output: {
       response: {
-        status: 500,
+        status: 403,
```

Comment:
will this be retried or aborted by rudder server and is that expected

Reply:
here auth category is `incative`..so these won't be retried